_compdemos/first_rhino.md

```diff
@@ -138,8 +138,7 @@ A few extra pieces of information for new users:
 choose appropriate selections to suit the needs of the task.
 If you aren't sure you need more than one CPU,
 you almost certainly do not.
-- There is not currently a way to execute `grabnode` with flags;
-you are required to use the interactive questions.
+- `grabnode` will take some common `sbatch` options and flags; contact `scicomp` if you need options beyond those offered by `grabnode` prompts.
```

Comment:
So it's possible to run `grabnode` in a single command, without the interactive prompts?

Reply:
@atombaby

Reply:
grabnode will always prompt, but the logic is such that command line arguments to grabnode will be processed.

Maybe this doc isn't the right place for this information at all...

Reply:
Good call. Will make a few adjustments

Reply:
I moved the note about flags to the main page describing `grabnode`, and made a few more formatting changes.